src/System.Net.Security/src/System/Net/SecureProtocols/AuthenticatedStream.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace System.Net.Security
+{
+    // A public contact for a base abstract authenticated stream.
+    public abstract class AuthenticatedStream : Stream
+    {
+        private Stream _InnerStream;
+        private bool _LeaveStreamOpen;
```

Comment:
I thought the tooling will automatically do that. I'll run it again. (@pgavlin found another place where style wasn't updated.)

I'll open bugs if these files are not changed and make the corrections manually